config/deploy/web.yaml.erb

```diff
@@ -139,6 +139,8 @@ spec:
                 key: elasticsearch_url
           - name: SEARCH_NUM_REPLICAS
             value: "<%= environment == 'production' ? 2 : 1 %>"
+          - name: SEARCH_INDEX_NAME
```

Comment:
This will be needed in all deployments like https://github.com/rubygems/rubygems.org/blob/master/config/deploy/jobs.yaml.erb.

Reply:
Good point, if we go with this solution we need to fix that first.